src/test/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheckTest.java

```diff
@@ -106,7 +136,24 @@ public void testRecordsAndCompactCtors() throws Exception {
         };
 
         verifyWithInlineConfigParser(
-                getNonCompilablePath("InputMethodLengthRecordsAndCompactCtors.java"),
+            getNonCompilablePath("InputMethodLengthRecordsAndCompactCtors.java"),
+            expected);
```

Comment:
please move this formatting out or revert it.

Reply:
done